libs/ledger-live-common/src/nft/support.ts

```diff
@@ -1,27 +1,35 @@
-import { Transaction, CryptoCurrency, ProtoNFT } from "../types";
+import { Transaction, CryptoCurrency, ProtoNFT, NFTStandard } from "../types";
 import { getEnv } from "../env";
 
-export const isNftTransaction = (transaction: Transaction): boolean => {
-  if (transaction.family === "ethereum") {
-    return ["erc721.transfer", "erc1155.transfer"].includes(transaction.mode);
+export const isNftTransaction = (
+  transaction: Transaction | undefined | null
+): boolean => {
+  if (transaction?.family === "ethereum") {
```

Comment:
Yup, agree. We should definitely have that rather than relying on our static typing. Here the 
```javascript 
if (transaction?.family === "ethereum")
```
 is enforced by typescript to make sure we have the property `mode` even though we're using optional chaining... 😮‍💨